src/hash_types.rs

```diff
@@ -52,7 +52,8 @@ hash_newtype!(WitnessMerkleNode, sha256d::Hash, 32, doc="A hash corresponding to
 hash_newtype!(WitnessCommitment, sha256d::Hash, 32, doc="A hash corresponding to the witness structure commitment in the coinbase transaction");
 hash_newtype!(XpubIdentifier, hash160::Hash, 20, doc="XpubIdentifier as defined in BIP-32.");
 
-hash_newtype!(FilterHash, sha256d::Hash, 32, doc="Bloom filter souble-SHA256 locator hash, as defined in BIP-168");
+hash_newtype!(FilterHash, sha256d::Hash, 32, doc="Filter hash, as defined in BIP-158");
+hash_newtype!(FilterHeader, sha256d::Hash, 32, doc="Filter header, as defined in BIP-158");
```

Comment:
Nit: `FilterHeader` is more elaborately defined in BIP157. Feel free to ignore as these two are often used interchangeably. Pointed out for completeness sake.